src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -346,6 +353,31 @@ def create(style_dataset, content_dataset, style_feature=None,
             else:
                 smoothed_loss = 0.9 * smoothed_loss + 0.1 * cur_loss
             iterations += 1
+
+            if params['checkpoint'] and iterations % params['checkpoint_increment'] == 0:
+                checkpoint_filename = params['checkpoint_prefixs'] + "-" + str(iterations) + ".model"
```

Comment:
Shouldn't the dict key be `'checkpoint_prefix'`, no trailing `'s'`?

Reply:
Yes, thank for the catch!